cpp/ql/test/query-tests/Likely Bugs/Likely Typos/ExprHasNoEffect/CMakeFiles/CMakeScratch/TryCompile-abcdef/CheckFunctionExists.c

```diff
@@ -0,0 +1,8 @@
+typedef long long size_t;
+
+size_t strlen(const char *s);
+
+int main() {
+  strlen("");
```

Comment:
It might be nice to have a comment somewhere explaining what's going on here.  i.e. this *would* produce a query result except that the file has been make to look like a CMake temporary file, so the result should be excluded.

Reply:
Something like this?
```suggestion
  strlen(""); // GOOD: the source file occurs in a `CMakeFiles/CMakeScratch/TryCompile-...' directory
```

Reply:
Yep, LGTM (I've committed it).